build.gradle

```diff
@@ -49,43 +36,76 @@ minecraft {
 }
 
 dependencies {
-	//to change the versions see the gradle.properties file
+	// To change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-
 	mappings "io.github.minecraft-cursed-legacy:plasma:b1.7.3-build.${project.plasma_build}"
-
 	modImplementation("io.github.minecraft-cursed-legacy:cursed-fabric-loader:${project.loader_version}") {
 		transitive false
 	}
 
 	// API. You technically don't need it, but it's extremely useful for not having to write the same code in every mod.
 	modImplementation "io.github.minecraft-cursed-legacy:cursed-legacy-api:${project.api_version}"
+
+	// PSA: Some older mods, compiled on Loom 0.2.1, might have outdated Maven POMs.
```

Comment:
Is this even applicable? We still use a modified version of Loom 0.2.

Reply:
Loom 0.2.x doesn't follow SemVer, each patch version increment in 0.2 should instead be considered as a minor version increment for practical purposes

Reply:
Oh yikes. OK whatever.